cmd/kube-controller-manager/app/controllermanager.go

```diff
@@ -532,16 +534,16 @@ func StartControllers(ctx ControllerContext, startSATokenController InitFunc, co
 		time.Sleep(wait.Jitter(ctx.ComponentConfig.Generic.ControllerStartInterval.Duration, ControllerStartJitter))
 
 		klog.V(1).Infof("Starting %q", controllerName)
-		debugHandler, started, err := initFn(ctx)
+		ctrl, err := initFn(ctx)
+		if err == cmerrors.ErrNotEnabled {
```

Comment:
Quibble: I would nest this if block inside the following if err != nil block. Primarily I think it's easier to read/reason about.

Reply:
That makes sense. It would be easier to read. Updating.